csharp/src/Google.Protobuf/Collections/RepeatedFieldExtensions.cs

```diff
@@ -0,0 +1,53 @@
+#region Copyright notice and license
+// Protocol Buffers - Google's data interchange format
+// Copyright 2015 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+#endregion
+
+using System;
+
+namespace Google.Protobuf.Collections
+{
+    /// <summary>
+    /// A set of extension methods on RepeatedField
```

Comment:
nit
```suggestion
    /// A set of extension methods on <see cref="RepeatedField{T}" />.
```
I don't remember if this works or not when there isn't the generic type in the context.